### PR TITLE
New world test

### PR DESCRIPTION
**Pull request by TinyGobby**
Jest testing added.
Frontend for signing in
Routes added

### CODE REVIEW

**Comment by TinyGobby** [comment]
[Add real name to form and tests](https://trello.com/c/OFnVGKyH/55-add-real-name-to-form-and-tests)

**Comment by TinyGobby** [comment]
[Jest Testing](https://trello.com/c/oNKq2voE/41-jest-testing)

File: app.test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  browser.close();
+})
+
+describe("Guess Vu", () => {
+  test('loads h1 with Guess Vu', async () => {
+    await page.waitForSelector('.App');
+    const html = await page.$eval('.App-title', e => e.innerHTML);
+    expect(html).toBe('Guess Vu!');
+  });
+  test('form works', async () => {
```
Comment (ckvuby) [inline app.test.js]: This does not test anything at the moment, looks like missing an expect.

File: frontend/src/form.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+
+class Form extends Component {
```
Comment (IvyMic) [inline frontend/src/form.js]: I think this should be a functional component because it doesnt hold any state.

**Pull request merged**